### ISSUE TITLE
YEN when create DB has bug

### ISSUE THREAD

**Issue by bluems** [YouPHPTube/YouPHPTube-Encoder-Network#12]
DB: MariaDB 5.5

I made db user 'tube-network'
but, YEN when installing time show message 'Error creating database: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-network' at line 1'

However, another program or SQL(e.g. YouPHPTube Encode and phpMyAdmin) can use that '-encoder' or '-network'

**Comment by DanielnetoDotCom** [comment]
cant you just use another database name?

**Comment by bluems** [comment]
> cant you just use another database name?

Of course, we've completed the installation under a different name and faced another bug(Do not log in and do not connect to more than one server).

However, this does not mean that this does not matter.
There is no problem with the DB, so you have to clarify what caused the bug.
Otherwise, bugs may occur even if one use different names.
YEN has a problem with SQL string parsing.
